src/pages/IstioConfigDetails/IstioObjectDetails/VirtualServiceRoute.tsx

```diff
@@ -258,9 +178,9 @@ class VirtualServiceRoute extends React.Component<VirtualServiceRouteProps> {
       tooltipFunction: () => {
         const badges = this.renderDestination(destinationWeight.destination);
         return (
-          <Tooltip id={`${u}_${destinationWeight.weight}`} key={`${u}_${destinationWeight.weight}`}>
+          <TooltipPF3 id={`${u}_${destinationWeight.weight}`} key={`${u}_${destinationWeight.weight}`}>
             <div className="label-collection">{badges}</div>
-          </Tooltip>
+          </TooltipPF3>
         );
```

Comment:
I don't udnerstand why you didn't migrate this Tooltip, second step ?

Reply:
Because It was needed in the bulletchart. And bulletchart is not migrated, pending of Patternfly upgrade. Since the PR is merged now, I can migrate that too.